packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -293,58 +296,87 @@ class RadioListTile<T> extends StatelessWidget {
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
   final bool? enableFeedback;
 
+  @override
+  State<RadioListTile<T>> createState() => _RadioListTileState<T>();
+}
+
+class _RadioListTileState<T> extends State<RadioListTile<T>> with TickerProviderStateMixin, ToggleableStateMixin {
```

Comment:
@HansMuller 
states from `states..add(MaterialState.selected);`   is part of the `ToggleableStateMixin` hence I have changed to class to Stateful

Reply:
https://github.com/flutter/flutter/pull/97972#discussion_r801992173

Why not just construct the states from the widget's selected parameter?

Reply:
That makes sense